### ISSUE TITLE
Login does not work with self-signed certificate

### ISSUE THREAD

**Issue by klawdhfzasjhaa** [stefan-niedermann/nextcloud-notes#459]
I have a LAN only setup with a self-signed cert. Login fails with message: "Anmeldung fehlgeschlagen. Ist die Notes-App auf dem Server aktiviert?"
I've read through old and active login issues, but couldn't find solution yet. NC server log also does not contain useful info about the problem.

Is the Notes app installed and enabled? Yes
Have you imported your self-signed cert into android keystore? Yes
Does login in Android browser work? Yes
Does login in Android Nextcloud app work? Yes
Does DAVDroid synchronisation work? Yes
Default port? Yes

Tasks: 0.9.6
Nextcloud: 13.0.5
nextcloud-notes: 0.21.0

**Comment by klawdhfzasjhaa** [comment]
Correct, before I imported the cert into the Android key store, I was asked to confirm. I did, but the login did not proceed. That's why I deleted the app data and retried with the cert **in** the key store. Also without success.

**Comment by schaarsc** [comment]
If you see the notification and you have confirmed the certificate, there should be no problems with TLS anymore. 
I suggest to revisit this issue after #418

**Comment by stefan-niedermann** [comment]
Closing in favor of #418 .
